library/pcs_property.py

```diff
@@ -116,11 +116,25 @@ def run_module():
     if cib_file is not None and os.path.isfile(cib_file):
         module.params['cib_file_param'] = '-f ' + cib_file
 
+    # get the pcs major.minor version
+    rc, out, err = module.run_command('pcs --version')
+    if rc == 0:
+        pcs_version = out.split('.')[0] + '.' + out.split('.')[1]
+    else:
+        module.fail_json(msg="pcs --version exited with non-zero exit code (" + rc + "): " + out + err)
+
```

Comment:
These codes are copied from below, but I don't know how to make the import succeed when it was separated into separate files as a common function.
https://github.com/OndrejHome/ansible.pcs-modules-2/blob/990400a6c0656c78db32ac5f5a715224e30b1bfc/library/pcs_auth.py#L105-L110